rgw/admin/user.go

```diff
@@ -46,6 +48,12 @@ type UserKeySpec struct {
 	SecretKey string `json:"secret_key" url:"secret-key"`
 }
 
+type userStat struct {
+	Size       uint64 `json:"size"`
```

Comment:
```suggestion
	Size       *uint64 `json:"size"`
```

Let's use pointers so don't get a built-in default value in the response.